### PR TITLE
Fix #290

### LINKED ISSUE #290: gui连接远程私链会异常

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo-gui/issues/290

### CODE REVIEW

File: neo-gui/UI/MainForm.cs [hunk 1]
```diff
@@ -260,8 +260,12 @@ private void timer1_Tick(object sender, EventArgs e)
...
             else
             {
-                toolStripProgressBar1.Value = persistence_span.Seconds;
-                toolStripProgressBar1.Style = ProgressBarStyle.Blocks;
+                try
+                {
+                    toolStripProgressBar1.Value = persistence_span.Seconds;
+                    toolStripProgressBar1.Style = ProgressBarStyle.Blocks;
+                }
+                catch { }
```
Comment (erikzhang) [inline neo-gui/UI/MainForm.cs]: Why will these lines throw exceptions?

Comment (shargon) [reply neo-gui/UI/MainForm.cs]: I you set the value higher than the max or lower than the min, it throws an exception

Comment (erikzhang) [reply neo-gui/UI/MainForm.cs]: But how can it be higher or lower? We have these lines:

https://github.com/neo-project/neo-gui/blob/3c4ae9a04e5cede1c89313d7da0d4d32eef5deef/neo-gui/UI/MainForm.cs#L255-L260

Comment (shargon) [reply neo-gui/UI/MainForm.cs]: Agree, should be great to have what value he obtains

**Review by lock9 — COMMENTED** [review]
This is treating the "symptom" but not the disease, shouldn't we try to find the root cause of this issue?
I can approve this issue if you guys agree this is acceptable since we are rebuilding this project later this year.

File: neo-gui/UI/MainForm.cs (hunk 1 above)
Comment (Ashuaidehao) [reply neo-gui/UI/MainForm.cs]: The toolStripProgressBar1's max value was set to 15:
https://github.com/neo-project/neo-gui/blob/master/neo-gui/UI/MainForm.Designer.cs#L558

If I config a private chain with persist block per 20 seconds , persistence_span "16" may enter the "else" clause and cause an exception.

Comment (shargon) [reply neo-gui/UI/MainForm.cs]: Thanks for all!

**Review by erikzhang — DISMISSED** [review]

**Comment by erikzhang** [comment]
Conflicts.

**Review by erikzhang — APPROVED** [review]

**Pull request merged**